free/src/test/scala/cats/free/FreeTests.scala

```diff
@@ -189,7 +189,7 @@ class FreeTests extends CatsSuite {
     forAll { (x: Int, y: Int) =>
       val expr1: Free[T, Int] = Free.injectRoll[T, Test1Algebra, Int](Test1(x, Free.pure))
       val expr2: Free[T, Int] = Free.injectRoll[T, Test2Algebra, Int](Test2(y, Free.pure))
-      val res = distr[T, Int](expr1 >> expr2)
+      val res = distr[T, Int](expr1 >>= (_ => expr2))
```

Comment:
Why won’t `*>` work here?

Reply:
There was something odd going on with the tests so that simply replacing >> with *> did not work.